.env

```diff
@@ -5,8 +5,8 @@ WORDPRESS_DB_USER=wordpress
 WORDPRESS_DB_PASSWORD=wordpress
 
 # WordPress CLI environment
-WORDPRESS_PORT=8084
-WORDPRESS_BASE_URL=http://localhost
+WORDPRESS_PORT=8889
+WORDPRESS_BASE_URL=http://localhost:8889
```

Comment:
any specific reason for this port?

Reply:
It's the default set for the test environment by `wp-env`